GOVERNANCE.md

```diff
@@ -0,0 +1,93 @@
+
+The **Board of Maintainers** is the main decision body of the UP project and is composed of all the current Maintainers.
+
+
+# Communication Channels
+
+We establish two official communication channels for the UP project.
+
+- The **Maintainers Mailing List** (unified-planning-maintainers@googlegroups.com): all and only the active Maintainers are part of this list. The mailing list is not publicly readable but anyone can send messages to it. It is the main and official communication channel among maintainers and to communicate to the board of maintainers.
+
+- The **Public Mailing List** (unified-planning@googlegroups.com): is a general-purpose communication channel to be used for both technical matters and public announcements. the list is publicly readable and anyone can join it. All decisions and votes of the Board of Maintainers are publicly communicated through this list.
+
+On these channels, as well as in any other virtual space associated with the library, the code of conduct of the UP project applies.
+
+
+# Technical Rules
+
+The following rules apply for the technical day-to-day governance of the project.
+
+- Any Maintainer can approve and commit to "master" a Pull-Request from any Committer, External Contribution or **another** Maintainer.
+
+- It is forbidden to anyone to self-approve and commit a Pull-Request.
+
+- It is mandatory for any commit pushed to master (after June 30th 2023) to be signed off to signal the acceptance of the Developer Certificate of Origin (DCO) rules.
+
+- It is forbidden to anyone to modify this document, unless **all** the Maintainers approve the Pull-Request.
+
+- Any Maintainer can make new releases (on GitHub and PyPI) of the UP library (taking the code of the "master" branch) autonomously, but must inform all the other Maintainers and Committers through the public mailing list.
+
+- Only a full consensus vote (meaning that all the Maintainers agree, with no abstentions) of the Board of Maintainers can change the name, web address or digital identity of the UP project.
+
+
+# Membership Rules
+
+The following rules govern the promotion and demotion of Committers and Maintainers.
+
+- Any Maintainer can autonomously nominate (and give the needed permissions) any number of Committers. (The rationale is that being a Committer does not grant any specific power over the project, but simplifies the work for people who often contribute to the library).
+
+- Any Maintainer can step down (effective immediately) from being a Maintainer by informing the Board of Maintainers.
+
+- Only a majority vote of the Board of Maintainers can nominate new Maintainers. The number rof Maintainers is not fixed.
+
+- Only a full consensus vote (meaning that all the Maintainers agree, with no abstentions) of the Board of Maintainers can modify the governance rules layed out in this document.
```

Comment:
This is already mentioned in the technical rules, but I think it still makes sense to repeat it here because it is not only a technical issue but this explicit statement makes clear that the board of all maintainers has the right to change the governance rules.